src/service/iot_config.proto

```diff
@@ -348,6 +348,14 @@ message load_region_req_v1 {
 
 message load_region_res_v1 {}
 
+message gateway_location_req_v1 {
+  bytes gateway = 1;
+  uint64 timestamp = 2;
```

Comment:
Why timestamp ? This is not a mutating call

Reply:
avoid replays

Reply:
this is just an informational "getter" correct? is there a replay attack vector with that?

Reply:
well we just don't want anybody to hit the API? otherwise I don't care that much

Reply:
Will remove